uint8 NO_EVENT = 0
uint8 MOUSE_EVENT_MOVE = 1
uint8 MOUSE_EVENT_PRESS = 2
uint8 MOUSE_EVENT_RELEASE = 3
uint8 KEYBOARD_EVENT_PRESS = 4
uint8 KEYBOARD_EVENT_RELEASE = 5

Header header
uint32 mouse_x
uint32 mouse_y
uint8  mouse_left
uint8  mouse_right
int16  btncode
uint8  type
